dallinger/recruiters.py

```diff
@@ -145,6 +154,26 @@ def recruit_participants(self, n=1):
             duration_hours=self.config.get('duration')
         )
 
+    def notify_recruited(self, participant, experiment):
+        """Assign a Qualification to the Participant for the experiment ID,
+        and for the configured group_name, if it's been set.
+        """
+        score = '1'  # always '1'; doesn't matter
```

Comment:
I meant to add that the main question is whether to do all of this as part of the current story, or create a new story to support the incrementing stuff.